tests/bsim/bluetooth/host/adv/chain/prj_extra_chain.conf

```diff
@@ -0,0 +1,41 @@
+CONFIG_BT=y
+CONFIG_BT_BROADCASTER=y
+CONFIG_BT_OBSERVER=y
+CONFIG_BT_EXT_ADV=y
+CONFIG_BT_EXT_ADV_MAX_ADV_SET=2
+#
+# The extra long device name requires chaining an additional PDU in this test-case
+CONFIG_BT_DEVICE_NAME="Broadcaster Multiple with an extra long name 12345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890"
```

Comment:
Wouldn't it have been "easier" to define some long advertising data in the test instead? The host may truncate this

Reply:
My point is that the name has specific handling in the host: https://github.com/zephyrproject-rtos/zephyr/blob/main/subsys/bluetooth/host/adv.c#L511-L518

Where it may be truncated, so using the name to set large advertising data may not work as intended. 

Any other advertising type would work ;)

Reply:
Removed the long name, and duplicated the manufacturer data field